### ISSUE TITLE
Rethink the necessity of on-rails/fixed camera angles

### ISSUE THREAD

**Issue by boarnoah** [Emseers/mouse2015#2]
The more I think about this the less sense a camera that follows rails makes sense to me. Since we are already thinking about situations where the path may have to split in two (at which point the game isn't strictly linear), what is the necessity of the fixed camera? 
Batman's use of the camera was restricted solely for the cinematic run sequence, not sure how well this would translate well into the full game's main form of camera, especially as the game involves solving a fair bit of puzzles.

Starting to think a camera that follows the player, that is user controlled (i.e camera follows the player's pivot, camera itself can be rotate (by say holding down the middle mouse and rotating/arrow keys). 

Because it's hard to justify fixed camera angles other than for the sake of having it imo. 

In conclusion, there is generally 2 kinds of 3rd person cameras, the behind the back one (think batman, asscreed etc...), and the follow and rotate camera (mmos, spectate cams in most games, etc...), not sure what we gain from straying to fixed methods.

**Comment by right-mouse** [comment]
Agreed. I thought about this and decided we should explore other options. I think it'll be a good idea to have this "on-rails camera" for short, fast-paced sequences where it will be more appropriate (imo). We'll have to discuss what type of camera to use for the main part of the game. For now, I'll rename the script to something else, so cameraMovement.cs can be used for the main camera (whatever we decide it to be).
